scripts/insert_into_appointments.sql

```diff
@@ -1 +1,12 @@
 -- insert appointments creating connections between doctors and patients
+
+
+-- I'm confused as to how I'm supposed to figure out the ID of the docotrs and
+-- patients without explicitly looking at the tables. If these tables were long
+-- in a real life situation, this would be extremely tedious.
+
+INSERT INTO appointments(app_date, app_time, doctor_id, patient_id, comment) VALUES
+  ('', ?, ?, 'comment'),
+  ('', ?, ?, 'comment'),
+  ('', ?, ?, 'comment'),
+  ('', ?, ?, 'comment');
```

Comment:
Yes, you're right - looking up the ID is NOT the solution. Instead, you could write some code to figure it out, like this:

```sql
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT '20150812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hilton'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Talley')
  UNION
  SELECT '20150817', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
   (SELECT id FROM doctors WHERE surname = 'Hilton'),
   (SELECT id FROM patients WHERE surname = 'Hodge')
;
```